virt/virt-4-8-release-notes.adoc

```diff
@@ -170,3 +170,6 @@ data:
 ----
 $ ovirt-aaa-jdbc-tool user unlock admin
 ----
+
+// fix targeted for 4.8.1
+* Importing a RHV VM fails if the affinity policy is `Migratable` even when live migration is enabled in {VirtProductName}. VM import succeeds if the affinity policy is `Pinned`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977277[*BZ#1977277*])
```

Comment:
This might just be my inexperience with how people pronounce it, but I would have expected it to be "an RHV", like pronouncing it "an R-H-V" spelled out. But that's fine if it's right as is!

Reply:
We pronounce it "Rev"